boards/posix/native_posix/doc/board.rst

```diff
@@ -168,6 +168,23 @@ available options::
      [-no-rt]                  :Do NOT slow down the execution to real time, but
                                 advance Zephyr's time as fast as possible and
                                 decoupled from the host time
+     [-rt-drift=<drift>]       :In parts per 1, drift of the simulated clock
```

Comment:
is ``parts per 1`` right?  Not sure what that means.

Reply:
You are right that this is not too clear. I meant similarly to how parts per thousand, or parts per million, are scaled by 1e6 or 1e3, this value is not scaled. So if somebody wants to write 10ppm, they should write it as 10e-6.

Reply:
How would you phrase that?